psuutil/main.py

```diff
@@ -182,11 +183,14 @@ def status(index, textonly):
             msg = 'NOT PRESENT'
         status_table.append([psu_name, msg])
 
+    if not status_table:
+        return
+
     if textonly:
```

Comment:
Now that table format is simplified, do we need a "textonly mode"?

Reply:
No, it doesn't needed because of SNMP uses PSU plugin directly. 
Removed "textonly mode